docs/pages/base-ui/getting-started/customization.js

```diff
@@ -1,7 +1,12 @@
 import * as React from 'react';
-import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocsV2';
+import AppFrame from 'docs/src/modules/components/AppFrame';
 import * as pageProps from 'docs/data/base/getting-started/customization/customization.md?@mui/markdown';
 
 export default function Page() {
   return <MarkdownDocs {...pageProps} />;
 }
+
+Page.getLayout = (page) => {
+  return <AppFrame>{page}</AppFrame>;
+};
```

Comment:
What I believe we should do on ALL the pages of the docs.

Reply:
@alexfauquette I have tested https://www.notion.so/mui-org/Migrate-all-pages-to-Page-getLayout-212d920a9ab245f08ac3a2aea874f1bc with a few more pages. I had to fix 2 bugs in `MarkdownDocsV2` (ad & codeblocks) but so far so good. The improvement is quite noticeable (before -> after):

<img width="320" alt="Screenshot 2023-07-19 at 22 56 04" src="https://github.com/mui/material-ui/assets/3165635/cfa5430d-71d2-4759-8340-1549b77c7bfb">

<img width="308" alt="Screenshot 2023-07-19 at 22 54 54" src="https://github.com/mui/material-ui/assets/3165635/b01e2862-b978-4d7b-828c-7964fc2b748f">

But it won't be enough to solve https://www.notion.so/mui-org/Docs-page-transition-performance-regression-b9e887541f5e4380b991ba173f4c0f6e though/